php-homework/h.w4.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+function primeNumber($n){
+
+    $x = 2;
+    $z = 0;
+    $y = 1;
+
+    while($x < $n){
+        $z = $n % $x;
+        ++$x;
+
+        if ($z == 0){
+            $y = 0;
+            break;  
+  
+        }
+    }
+   
+    if($y == 0 || $n == 0 || $n == 1){
```

Comment:
You implementation is already correct, but you should handle special cases first to avoid executing unnecessary source code.
You can check hint at https://gitlab.com/pascaliaasia/pav-training/-/blob/basic-training/basic-programing/php/homework-hint/php-hint.md